app/views/docs/databases.phtml

```diff
@@ -107,33 +107,45 @@
         <div class="ide" data-lang="javascript" data-lang-label="Web SDK">
             <pre class="line-numbers"><code class="prism language-javascript" data-prism>import { Client, Databases } from "appwrite";
 
-const client = new Client();
-
-client
+const client = new Client()
     .setEndpoint('https://[HOSTNAME_OR_IP]/v1')
     .setProject('[PROJECT_ID]');
 
-const databases = new Databases(client, '[DATABASE_ID]');
+const databases = new Databases(client);
 
-const promise = databases.createDocument('[COLLECTION_ID]', 'unique()', {});
+const promise = databases.createDocument(
+    '[DATABASE_ID]',
+    '[COLLECTION_ID]',
+    ID.unique(),
+    {}
+);
 
 promise.then(function (response) {
-    console.log(response); // Success
+    console.log(response);
 }, function (error) {
-    console.log(error); // Failure
+    console.log(error);
 });</code></pre>
         </div>
     </li>
     <li>
         <h3>Flutter</h3>
         <div class="ide" data-lang="dart" data-lang-label="Flutter SDK">
             <pre class="line-numbers"><code class="prism language-dart" data-prism>import 'package:appwrite/appwrite.dart';
+
 void main() async {
-    final client = Client();
-    final databases = Databases(client, databaseId: '[DATABASE_ID]');
+    final client = Client()
+        .setEndpoint('https://[HOSTNAME_OR_IP]/v1')
+        .setProject('[PROJECT_ID]');
+
+    final databases = Databases(client);
+
     try {
-        final doc = databases.createDocument(collectionId: '[COLLECTION_ID]', documentId: 'unique()', data: {});
-        print(doc.toMap());
+        final document = databases.createDocument(
+            databaseId: '[DATABASE_ID]',
+            collectionId: '[COLLECTION_ID]',
+            documentId: ID.unique(),
+            data: {}
```

Comment:
Can we have a consistent and meaningful example for all of the placeholder data?

We can use our usual:
```js
data: {
  'title': 'The Lion King',
  'year': 1994
}
```

This also helps us avoid the mapof('a' to 'b') example, which is mildly confusing